saleor/stock/management.py

```diff
@@ -0,0 +1,42 @@
+from typing import TYPE_CHECKING
+
+from .models import Stock
+
+if TYPE_CHECKING:
+    from ..product.models import ProductVariant
+
+
+def allocate_stock(
+    variant: "ProductVariant", country_code: str, quantity: int, commit: bool = True
+) -> Stock:
+    stock = Stock.objects.for_country(country_code).get(product_variant=variant)
+    stock.allocate_stock(quantity, commit=commit)
```

Comment:
We should lock database when we are changing a quantity. `select_for_update` would be helpful